libraries/ESP_NOW/examples/ESP_NOW_Serial/ESP_NOW_Serial.ino

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
this example as-is will work only on specific two devices. I suggest the code be written to support only 1 device, who's MAC address you pre-define. Then the Serial can be defined as `ESP_NOW_Serial_Class NowSerial(mac, ESPNOW_WIFI_CHANNEL, ESPNOW_WIFI_IF);` which is easier than pointers